tests/phpunit/elementor/modules/lazyload/test-module.php

```diff
@@ -6,15 +6,15 @@
 
 class Elementor_Test_LazyLoad extends Elementor_Test_Base {
 
-	public function test_reduce_background_image_size() {
+	public function test_remove_background_image() {
 
 		//Arrange
 		$image_id = $this->factory()->attachment->create_upload_object( dirname( __DIR__, 3 ) . '/resources/mock-image.png' );
 		$image_url = wp_get_attachment_url( $image_id );
 
 		//Act
 		$reflection = new \ReflectionClass( LazyLoad::class );
-		$method = $reflection->getMethod( 'reduce_background_image_size' );
+		$method = $reflection->getMethod( 'remove_background_image' );
```

Comment:
Why use reflection? you can call the `hook`

Reply:
My code uses a function that exists in a 3rd-party plugin. It was for that reason that I had to use reflection.

Reply:
I just need to test `append_lazyload_selector` logic